tests/FunctionalResolverTest.php

```diff
@@ -38,9 +38,9 @@ public function testResolveAllLocalhostResolvesWithArray()
     /**
      * @group internet
      */
-    public function testResolveGoogleResolves()
+    public function testResolveBingResolves()
     {
-        $promise = $this->resolver->resolve('google.com');
+        $promise = $this->resolver->resolve('bing.com');
```

Comment:
I don't see why we should change `google.com` to `bing.com` here, seems unnecessary. Same goes for the other tests.

Reply:
In China,we can not visit google.com, so tests will not pass on my computer,But bing.com we can

Reply:
I agree with you that we should use something that works for everyone, in some of our projects we started using `reactphp.org`  instead, but I don't think this belongs in this pull request. Maybe use `bing.com` for testing purposes on your system without committing it in the end.

What do you think?

Reply:
@SimonFrings IMHO it makes sense to change everything to `reactphp.org` to align this across the board. Will file PoC PR for that to take this discussion there.

@Chrisdowson I assume you can reacht `reactphp.org`  just fine?

Reply:
> 

yes,I can reach `reactphp.org`

Reply:
> I agree with you that we should use something that works for everyone, in some of our projects we started using `reactphp.org` instead, but I don't think this belongs in this pull request. Maybe use `bing.com` for testing purposes on your system without committing it in the end.
> 
> What do you think?

I agree with WyriHaximus, How about I replace `bing.com` with `reactphp.org` now?

Reply:
@WyriHaximus @Chrisdowson  I agree with both of you, I am not sure if this is something we should tackle in this pull request as it doesn't fit to the other changes made in here and could seem a bit random. It also keeps the diff significantly smaller and lays focus on the actual changes. That's the reason I suggest a (follow-up) pull request where we change `google.com` to `reactphp.org`.